src/services/search.service.ts

```diff
@@ -5,15 +5,20 @@ import { CipherView } from '../models/view/cipherView';
 import { CipherService } from '../abstractions/cipher.service';
 import { SearchService as SearchServiceAbstraction } from '../abstractions/search.service';
 
+import { LogService } from '../abstractions/log.service';
 import { CipherType } from '../enums/cipherType';
 import { FieldType } from '../enums/fieldType';
 import { UriMatchType } from '../enums/uriMatchType';
+import { ConsoleLogService } from '../services/consoleLog.service';
```

Comment:
Should move the actual service import into its own group.....
Curious, after seeing so many of these; wouldn't the Angular app inject the appropriate log service, as long as each client has provided the init code to new up one and provide it? We don't follow this pattern with any other services where the constructor checks if null and creates otherwise; instead leaving the default instantiation to the overall app (main.ts, etc., class `Main`) and DI.

Reply:
Web and Browser never implement a ConsoleLogService This was a cheeky thing to try and avoid changing those repos. Since I need to add `browser-process-hrtime` to those repos, that obviously hasn't worked. 

I also think that this service is a bit of a special case. If I don't care about `console`, I'd rather not think about it than have to provide one explicitly. I can get angular to supply these if we want to go that route for congruity.

Reply:
Yes, let's have angular supply these through the normal course of DI and for congruity (nice word). Thanks Matt!